### PR TITLE
LRDOCS-8024 Migrate cache portal props to echcache XML files

### PR DESCRIPTION
**Pull request by jhinkey**
https://issues.liferay.com/browse/LRDOCS-8024

### CODE REVIEW

File: docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay-dxp/configuration-and-infrastructure/migrating-configurations-and-properties.md [hunk 1]
```diff
@@ -131,23 +131,25 @@ Put the `.config` files in a folder called `[Liferay Home]/osgi/configs`.
...
     ```
 
-```note::
-   You can build image sprites using any framework you like and deploy them in your plugins.
-```
+    ```note::
+       You can build image sprites using any framework you like and deploy them in your plugins.
+    ```
+
+1. As of 7.3, cache is solely configured using Ehcache XML files. Cache is no longer enabled or configured using portal properties. If you have configured cache using portal properties, configure the cache using Ehcache XML files in a module. Please see [Cache Configuration](https://help.liferay.com/hc/en-us/articles/360035581451-Introduction-to-Cache-Configuration) for more information.
```
Comment (jhinkey) [inline docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay-dxp/configuration-and-infrastructure/migrating-configurations-and-properties.md:152]: Added the migration instruction here.

File: docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay-dxp/configuration-and-infrastructure/migrating-configurations-and-properties.md [hunk 2]
```diff
@@ -131,23 +131,25 @@ Put the `.config` files in a folder called `[Liferay Home]/osgi/configs`.
...
 
-* Examine the default settings changes in 7.0+ [here](../reference/changes-to-default-settings.md).
+1. Examine the default settings changes in 7.0+ [here](../reference/changes-to-default-settings.md).
 
-* Liferay's image sprite framework is deprecated as of 7.2 and is disabled by default. The framework requires scanning plugins for image sprites. If you don't use the framework, there's no need for it to scan for images sprites. If you use the framework yourself, enable it by overriding the default `sprite.enabled` portal property (since 7.2) value with the following setting in your [`portal-ext.properties`](../../reference/portal-properties.md) file:
+1. Liferay's image sprite framework is deprecated as of 7.2 and is disabled by default. The framework requires scanning plugins for image sprites. If you don't use the framework, there's no need for it to scan for images sprites. If you use the framework yourself, enable it by overriding the default `sprite.enabled` portal property (since 7.2) value with the following setting in your [`portal-ext.properties`](../../reference/portal-properties.md) file.
```
Comment (jhinkey) [inline docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay-dxp/configuration-and-infrastructure/migrating-configurations-and-properties.md:142]: Changed these from unordered to ordered list, as numbers are easier for support to reference.

**Comment by sez11a** [comment]
Just started reviewing :)

**Comment by sez11a** [comment]
Superseded by https://github.com/jrhoun/liferay-learn/pull/577

**Pull request closed without merging**